### ISSUE TITLE
Spurious ;s may be confusing

### ISSUE THREAD

**Issue by Ndolam** [codewrite/arduino-capacitor#7]
In Capacitor.cpp you have many lines like
`#define STRAY_CAP (17.50);`

The ';' at the end should really not be there.  The program works in the face of this because, as of right now, the only place those defined tokens are used is right before another ';', and an empty statement is OK in those contexts.

But people looking at your code and modifying it might find themselves with weird problems, and non-C/C++ programmers might incorrectly think those ';'s are needed.

While I'm here, note that the ()s around the numbers are not needed, since the numbers are single tokens.
In summary all is needed are lines like
`#define STRAY_CAP 17.50`

Cheers.

**Comment by codewrite** [comment]
Fixed, Thanks

**Comment by Ndolam** [comment]
Thanks for the quick response.
